Frontend/Helpers/TagHelpers/GdsKeyValueTagHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Frontend.Helpers.TagHelpers
+{
+    [HtmlTargetElement("gds-key-value", TagStructure = TagStructure.NormalOrSelfClosing)]
+    public class GdsKeyValueTagHelper : TagHelper
+    {
+        public string Key { get; set; }
+        public string Value { get; set; }
+        
+        private readonly HtmlEncoder _htmlEncoder;
+
+        public GdsKeyValueTagHelper(HtmlEncoder htmlEncoder)
+        {
+            _htmlEncoder = htmlEncoder;
+        }
+        
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            output.TagMode = TagMode.StartTagAndEndTag;
+            output.TagName = "div";
+            output.AddClass("govuk-summary-list__row", HtmlEncoder.Default);
+
+            var dt = new TagBuilder("dt");
+            dt.AddCssClass("govuk-summary-list__key");
+            dt.InnerHtml.SetContent(Key);
+            
+            var dd = new TagBuilder("dd");
+            dd.AddCssClass("govuk-summary-list__value");
```

Comment:
Think the dd tag also has `dfe-summary-list__value--width-50` css class to split the columns evenly

Reply:
tbh it might be better to just put that in the actual html element rather than in the tag helper - not sure really whatever you think is best